charts/zora/values.yaml

```diff
@@ -175,6 +175,9 @@ scan:
       # -- Overrides the image tag whose default is the chart appVersion
       tag: ""
   plugins:
+    # -- Annotations added to the plugin service account
+    annotations:
+      service-account-annotation: service-account-value
```

Comment:
```suggestion
      #service-account-annotation: service-account-value
```

Reply:
I already tried this, but it caused the README to be generated as a string.  Leaving as is for now.